share/spack/gitlab/cloud_pipelines/.gitlab-ci.yml

```diff
@@ -264,9 +264,11 @@ e4s-mac-protected-build:
 
 e4s-pr-generate:
   extends: [ ".e4s", ".pr-generate"]
+  image: ecpe4s/ubuntu20.04-runner-x86_64:2022-05-01
```

Comment:
Would it make sense to move directly to 22.04 i.e. the latest LTS?

Reply:
Great idea! I will generate our 22.04 images today and update here!

Reply:
Although, is there concern that ROCm stack is not OK on Ubuntu 22.04 yet? @srekolam do you know?